src/pkg/build/conveyorPacker_debootstrap_test.go

```diff
@@ -48,19 +57,23 @@ func TestDebootstrapPacker(t *testing.T) {
 	testDef := Definition{}
 
 	testDef.Header = map[string]string{
-		"Bootstrap": "debootstrap",
-		"OSVersion": "bionic",
-		"MirrorURL": "http://us.archive.ubuntu.com/ubuntu/",
-		"Include":   "apt python ",
+		"bootstrap": "debootstrap",
+		"osversion": "bionic",
+		"mirrorurl": "http://us.archive.ubuntu.com/ubuntu/",
+		"include":   "apt python ",
 	}
 
 	dcp := DebootstrapConveyorPacker{}
 
 	err := dcp.Get(testDef)
 	if err != nil {
+		//clean up tmpfs since assembler isnt called
+		os.RemoveAll(dcp.tmpfs)
 		t.Fatalf("Debootstrap Get failed: %v", err)
 	}
 
+	defer os.RemoveAll(dcp.tmpfs)
```

Comment:
Similar comment here, move this `defer` right after the `dc.Get()` and remove the duplicate `os.RemoveAll()`.